.papr.yml

```diff
@@ -89,15 +89,17 @@ required: true
 host:
   distro: fedora/26/atomic
 
-# copy yum.repos.d to get any injected repos from the host, which
-# will point to a closer mirror
+# Copy yum.repos.d to get any injected repos from the host, which
+# will point to a closer mirror.  Note we substitute $releasever
+# since https://github.com/projectatomic/rpm-ostree/pull/875
 tests:
   - docker run --privileged --rm
     -e RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO=1
     -v /etc/yum.repos.d:/etc/yum.repos.d.host:ro
     -v $(pwd):/srv/code -w /srv/code
     registry.fedoraproject.org/fedora:26 /bin/sh -c
     "cp -fv /etc/yum.repos.d{.host/*.repo,} &&
+     for x in /etc/yum.repos.d/*.repo; do sed -i -e 's,\$releasever,26,' \$x; done && \
```

Comment:
You don't need the escaping backslash at the end of this line here. I have this link handy: https://stackoverflow.com/a/21699210/308136